StarCellBio/views.py

```diff
@@ -160,10 +172,14 @@ def get_student_courses(request, **kwargs):
     obj.domain='starcellbio.mit.edu'
     obj.save()
 
+
     if request.user.id and len(request.user.groups.all()) > 0:
         account_type = request.user.groups.all()[0].name
     else:
         account_type = ''
+# Ivan's*/
```

Comment:
remove, and in general, there are tons of commented out debug code and such, please just remove all of that